### PR TITLE
#2167 web3 suggestions are missing in Console chat

### LINKED ISSUE #2167: Web3 suggestions are missing in Console chat [develop]
### Description

*Type*: Bug

*Summary*: No suggestions shown when entering `web3` in console

#### Expected behavior
Suggestions should be displayed

#### Actual behavior
image

### Reproduction
- Open Status
- Go to Console
- Type `web3`

### Additional Information
* Status version: develop `0.9.10-238-g82363020+ (1795)`
* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by alwx**
Fixes #2167

### CODE REVIEW

**Review by janherich — APPROVED** [review]

File: src/status_im/chat/events/input.cljs [hunk 1]
```diff
@@ -64,28 +64,40 @@
...
+      (assoc-in [:chats current-chat-id :input-text] (input-model/text->emoji chat-text))
+
+      (and dapp? (str/blank? chat-text))
+      (assoc-in [:chats current-chat-id :parameter-boxes :message] nil))))
+
+(defn call-on-message-input-change
+  "Calls bot's `on-message-input-change` function"
+  [{:keys [current-chat-id current-account-id chats local-storage] :as db}]
+  (let [chat-text (-> (get-in chats [current-chat-id :input-text])
+                      (or "")
```
Comment (rasom) [inline src/status_im/chat/events/input.cljs]: Empty string can be passed as the last parameter in `get-in`

**Comment by alwx** [comment]
@rasom thanks for the suggestion. Fixed.

File: src/status_im/chat/events/input.cljs [hunk 2]
```diff
@@ -64,28 +64,39 @@
...
+      (assoc-in [:chats current-chat-id :input-text] (input-model/text->emoji chat-text))
+
+      (and dapp? (str/blank? chat-text))
+      (assoc-in [:chats current-chat-id :parameter-boxes :message] nil))))
+
+(defn call-on-message-input-change
+  "Calls bot's `on-message-input-change` function"
+  [{:keys [current-chat-id current-account-id chats local-storage] :as db}]
+  (let [chat-text (-> (get-in chats [current-chat-id :input-text] "")
+                      (str/trim))
```
Comment (rasom) [inline src/status_im/chat/events/input.cljs]: No need to use threading macro here btw

**Comment by annadanchenko** [comment]
I’m deleting the command from input field and got `null is not an object (evaluating e.trim)`. See logs and video in https://app.testfairy.com/projects/4803590-status/builds/6903249/sessions/2/?accessToken=WhX8yN5N-0KmPSBoDoxUO9T/dh0

**Comment by alwx** [comment]
The problem happened because `(get-in chats [current-chat-id :input-text] "")` is not the same as `(or (get-in chats [current-chat-id :input-text]) "")` :)
Fixed now.

**Comment by oskarth** [comment]
Tested OK according to https://github.com/status-im/status-react/issues/2167#issuecomment-338159435

Merging to develop

**Pull request merged**